.rubocop.yml

```diff
@@ -1,107 +1,23 @@
+require:
+  - rubocop-performance
+  - rubocop-rails
+  - rubocop-rspec
+  - rubocop-sorbet
+
 inherit_from: .rubocop_todo.yml
 
 AllCops:
+  NewCops: enable
```

Comment:
If we do this, we probably want to pin tightly to the patch level rubocop.  I think we've avoided this in other projects so we can autoupdate rubocop without breaking the rubocop check.

Reply:
@jcoyne as in, instead of `~> 0.90.0`, `0.90.0`?